### PR TITLE
Merge Xmpp systemd set-up

### PR DESCRIPTION
**Pull request by nivemaham**

### CODE REVIEW

File: dcompose-stack/radar-cp-hadoop-stack/bin/hdfs-restructure [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+OUTPUT_DIR=${2:-$DIR/output}
+OUTPUT_DIR="$(cd "$(dirname "$OUTPUT_DIR")"; pwd)/$(basename "$OUTPUT_DIR")"
+
+cd $DIR
+
+. .env
+
 # HDFS restructure version
-DOCKER_IMAGE=radarbase/radar-hdfs-restructure:0.5.3
+DOCKER_IMAGE=radarbase/radar-hdfs-restructure:0.5.7
```
Comment (blootsvoets) [inline dcompose-stack/radar-cp-hadoop-stack/bin/hdfs-restructure:19]: I recall there being some issue with 0.5.7, we can use 0.5.6 definitely.

File: dcompose-stack/radar-cp-hadoop-stack/travis/travis-env.template [hunk 2]
```diff
@@ -23,17 +23,18 @@ HDFS_DATA_DIR_2=/home/travis/data/hdfs-data-2
...
+POSTGRES_VERSION=10.6-alpine
 KAFKA_MANAGER_USERNAME=kafkamanager-user
 KAFKA_MANAGER_PASSWORD=kafkamanager-pwd
 PORTAINER_PASSWORD_HASH=$2y$05$POrvWxSUsnkeGNZ7LzHUceL6rQM3U79lYofaM2NtyQJ8iWJ4ve542
-MANAGEMENTPORTAL_COMMON_ADMIN_PASSWORD=notadmin
 MANAGEMENTPORTAL_OAUTH_CHECKING_KEY_ALIASES_0=radarbase-managementportal-ec
 MANAGEMENTPORTAL_OAUTH_CHECKING_KEY_ALIASES_1=selfsigned
+RESTRUCTURE_OUTPUT_DIR=output
+RESTRUCTURE_NUM_THREADS=3
+RESTRUCTURE_ENABLE_SNAPSHOTS=true
```
Comment (blootsvoets) [inline dcompose-stack/radar-cp-hadoop-stack/travis/travis-env.template:34]: We might want to set this to false for the time being.

**Review by blootsvoets — APPROVED** [review]
LGTM. Some minor comments.

**Comment by nivemaham** [comment]
Merged with conflicts resolved

**Pull request closed without merging**